src/tribler-gui/tribler_gui/core_manager.py

```diff
@@ -64,7 +64,12 @@ def on_core_read_ready(self):
         if b'Traceback' in raw_output:
             self.core_traceback = decoded_output
             self.core_traceback_timestamp = int(round(time.time() * 1000))
-        print(decoded_output.strip())  # noqa: T001
```

Comment:
Sometimes Tribler crashes when trying to write data to stdout during the shutdown